lib/plans/checkout.dart

```diff
@@ -473,4 +476,21 @@ class _CheckoutState extends State<Checkout>
       return false;
     }
   }
+
+
+  void hasPlansUpdateOrBuy() {
+    try {
+      retry(() async {
```

Comment:
I guess the delay between attempts is computed as `pow(2, attempt) * delayFactor` where delayFactor defaults to 200 milliseconds: https://pub.dev/documentation/retry/latest/retry/RetryOptions-class.html

"With the default configuration functions will be retried up-to 7 times (8 attempts in total), sleeping 1st, 2nd, 3rd, ..., 7th attempt:

400 ms +/- 25%
800 ms +/- 25%
1600 ms +/- 25%
3200 ms +/- 25%
6400 ms +/- 25%
12800 ms +/- 25%
25600 ms +/- 25%"

I think the max attempts is fine, but it might make sense to bump the delay factor to something like twenty seconds instead?

Reply:
I agree, I gonna make this change.